package.json

```diff
@@ -45,7 +45,6 @@
     "build": "run-p build:*",
     "build:project": "tsdx build",
     "build:bin": "tsc && rollup -c rollup.config.js",
-    "prepare": "tsdx build",
```

Comment:
@fabien0102 I think this caught out whoever did the publish of `9.8.0` so I've removed it from the scripts.

See: #222

Reply:
Correct, but this should be `npm run build` instead, so we are sure to not forget to build before publishing (oh yes, this happens so easily! 😁)

I will this directly on master before publishing 😉